apps/studio/components/interfaces/Storage/VectorBuckets/VectorBucketDetails.tsx

```diff
@@ -164,7 +213,7 @@ export const VectorBucketDetails = () => {
                                 >
                                   {/* TODO: Proper URL for table editor */}
                                   <Link
-                                    href={`/project/${projectRef}/editor/${encodeURIComponent(name)}?schema=${bucketId}`}
+                                    href={`/project/${projectRef}/editor/${encodeURIComponent(name)}?schema=${getVectorBucketFDWSchemaName(bucketId!)}`}
```

Comment:
not sure if this can be done within this PR but the URL needs to have the table ID, not the table name